docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+#. Immediately navigating back and trying to click "Start Exam" should't allow you to start the exam
+#. Navigating back to your proctoring software setup page and refreshing on that page should cause no issues
+#. Go through your proctoring software setup for your Webcam, Desktop, etc. Upon setup completion, you should be redirected back to an exam interstitial with just the "Start Exam" button
+#. Close your proctoring software and click "Start Exam". The exam should not start
+
+Exam Flow Works With Proctoring Software Installed & Running
+------------------------------------------------------------
+#. Go through the setup again, this time without closing your proctoring software. Clicking "Start Exam" should start the exam
+#. Complete whatever question you put inside your exam w/ the correct answer. Clicking "End My Exam" should direct you to a "Are you sure you want to end your proctored exam?" interstitial
+#. Clicking "No I'd like to continue working" should send you back to the exam
+#. Clicking "Yes end my proctored exam" should end the exam and directs you to a "You have submitted this proctored exam for review" page
+#. In the course view, the exam you took should be marked as completed (with a green checkmark)
+
+
+Proctorio-Specific Tests
+========================
+These tests are for those using Proctorio as their proctoring provider
+
+Exam Flow Works as Expected
+---------------------------
+#. With Proctorio not installed, clicking "Start System Check" redirects you to https://getproctorio.com/
+#. Go through the Proctorio Setup for your Webcam, Desktop, etc. Upon setup completion, your webcam should appear to the right of the screen along with some controls. 
+#. On the "Return to exam" page, DO NOT click on the "Sending you back to your exam" page. Wait instead, and you should be sent to the "Important"/"Start Exam" interstitial automatically
+#. In a separate exam, go through the setup again and on the "Sending you back to your exam." page, click "Return to Exam". Clicking this link should send you to another interstitial. (NOTE: Clicking this link is time-bound so make sure to click it fast)
+#. Clicking the "Return to Exam" button should direct you to a page titled "Important" w/ another header saying "Proctored Exam Rules", and a "Start Exam" Button
+
+
+Exams Dashboard
+===============
+Expected Behavior: Calls to the ACS Endpoint work, instructors can review/reset exam attempts, and expected downstream effects (e.g. w/ completion, credits, certificates, etc) of modifying an exam attempt's status occur
+NOTE: These instructions currently do not account for downstream effects involving a learner's credit requirement status. We currently have no plan to include these
+
+Dashboard UI Loads Exam Attempts
+--------------------------------
+#. Clicking on the "Special Exams" tab should load the *new* version of the exams dashboard
```

Comment:
Where is this tab? How do I get to this tab? How do I know if I'm at the old or new exams dashboard?

Reply:
Added instructions on how to get to this tab + how to tell if it's the newer one.